lib/infoWidget.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'characterModel.dart';
+
+class InfoWidget extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final Character character = ModalRoute.of(context).settings.arguments;
+    return Center(
+      child: Column(
+        mainAxisSize: MainAxisSize.max,
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          Padding(
```

Comment:
Create a new custom widget for example CharacterProperty, for all these same Padding Text Style widgets